control-plane/csi-driver/src/bin/node/dev/nvmf.rs

```diff
@@ -192,12 +193,17 @@ impl Attach for NvmfAttach {
                     .hostnqn(self.hostnqn.clone())
                     .keep_alive_tmo(self.keep_alive_tmo)
                     .build()?;
-                match ca.connect() {
-                    // Should we remove this arm?
-                    Err(NvmeError::ConnectInProgress) => Ok(()),
-                    Err(err) => Err(err.into()),
-                    Ok(_) => Ok(()),
-                }
+
+                spawn_blocking(move || {
+                    match ca.connect() {
```

Comment:
Now that this is not blocking, what happens if the future is dropped but the connect/disconnect ioctls are still in progress.
Maybe we should issue these by non-cancellable futures (worker-threads)? Otherwise each retry will stack them up (probably can be another PR)

Reply:
What happens if the join handle is dropped?